rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+DRIVERS=="cmpc_accel_v4", \
```

Comment:
> It matches both input8 and event7.

Yeah, SUBSYSTEMS walks up the chain, so you'd have matches for both.

> When processing input8, input8 has a modalias input:b0000v0000p0000e0000-e0,3,kra0,1,2,mlsfw which is used for $attr{modalias} so that is looked up in hwdb with no match, so no matrix applied.

Here it walks up the chain as well, and will hit the first `modalias` when dealing with the input device (which is the device itself, or the parent device, depending on which match we're processing).

You'll want to match with:
`${attr[XXX]modalias}`
where `XXX` is an attribute that's unique to the ACPI device.

I think `hid` might be what we're looking for. You should end up with both `input8` and `event7` having mount matrices. Ideally, only `input8` would, but we'd need to tighten the match rules for that. Maybe add a match on the `DEVNAME` being empty?